examples/cloudflare-durable-objects.md

```diff
@@ -1,48 +1,146 @@
 # Cloudflare Durable Objects
 
-By using Hono, you can write [Durable Objects](https://developers.cloudflare.com/durable-objects/) application easily.
-
-Hono can handle a `fetch` event of Durable Objects and you can use it with the powerful router.
-
-```ts
-import { Hono } from 'hono'
-
-export class Counter {
-  value: number = 0
-  state: DurableObjectState
-  app: Hono = new Hono()
-
-  constructor(state: DurableObjectState) {
-    this.state = state
-    this.state.blockConcurrencyWhile(async () => {
-      const stored = await this.state.storage?.get<number>('value')
-      this.value = stored || 0
-    })
-
-    this.app.get('/increment', async (c) => {
-      const currentValue = ++this.value
-      await this.state.storage?.put('value', this.value)
-      return c.text(currentValue.toString())
-    })
-
-    this.app.get('/decrement', async (c) => {
-      const currentValue = --this.value
-      await this.state.storage?.put('value', this.value)
-      return c.text(currentValue.toString())
-    })
-
-    this.app.get('/', async (c) => {
-      return c.text(this.value.toString())
-    })
+You can use Hono as the router in your Cloudflare Worker, calling RPC (Remote Procedure Calls) to interact with [Durable Objects](https://developers.cloudflare.com/durable-objects/). This is the recommended approach as of Cloudflare Workers compatibility date `2024-04-03`.
+
+## Example: Counter Durable Object
+
+### Initial Setup
```

Comment:
I think you don't need to put the document for general implementation not using Hono since the users can know it from the Cloudflare document. And I don't want to use `create-cloudflare` as much as possible. If the command menus are changed, we have to update the docs.

This means you can remove the `## Example: Counter Durable Object` section and add just an example using Hono with `Counter`.